app/templates/patch.hbs

```diff
@@ -2,6 +2,7 @@
   <div class="file">
     <div class="meta">
       <span>{{filePath}}</span>
+      <a class="target" {{bind-attr href=targetFileUrl}} title="View generated file at target version">View</a>
```

Comment:
I know it's silly, but for completeness, I think I would rather have links to the source & target file. "View" obviously wouldn't work as the text for both links, using the version numbers is probably sufficient.